### ISSUE TITLE
Typescript definitions are wrong

### ISSUE THREAD

**Issue by duro** [launchdarkly/node-server-sdk-dynamodb#10]
The current TypeScript definitions throw type errors when the package is imported.

When imported as such:

```
import DynamoDBFeatureStore from 'launchdarkly-node-server-sdk-dynamodb'
```

The following type error is emitted from the compiler:

```
Cannot invoke an expression whose type lacks a call signature. Type 'typeof import("launchdarkly-node-server-sdk-dynamodb")' has no compatible call signatures.
```

This is because the default import has not been properly tagged, and so typescript thinks that DynamoDBFeatureStore is a named import and wants you to import like so:

```
import { DynamoDBFeatureStore } from 'launchdarkly-node-server-sdk-dynamodb'
```

But then at run time, this throws an error because that named import does not actually exist.

**Comment by eli-darkly** [comment]
Thanks for catching this. The fix looks good, and I see that we made the same mistake in another similar project (the Consul integration), so I'll fix that one.

**Comment by eli-darkly** [comment]
I'm also adding a TypeScript compilation test that would have caught this (i.e. trying to compile a TS file that uses the export). We already have a step for that, but we weren't actually using it.

**Comment by eli-darkly** [comment]
Fix was released in 1.1.6.
